docs/content/latest/explore/query-1-performance/explain-analyze.md

```diff
@@ -55,7 +55,7 @@ To insert table rows, execute the following:
 
 ```sql
 yugabyte=# INSERT INTO employees (k1, k2, v1, v2) 
-  VALUES (1, 2.0, 3, 'a'), (2, 3.0, 4, 'b'), (3, 4.0, 5, 'c');
+VALUES (1, 2.0, 3, 'a'), (2, 3.0, 4, 'b'), (3, 4.0, 5, 'c');
```

Comment:
I had a few failed executions because of the spaces, so I decided to remove whatever I can to be on the safe site (I don't know how the page will render on different browsers and what the "copy" action can bring into the user's console)